packages/gcf-utils/src/bin/README.md

```diff
@@ -47,6 +47,7 @@ Application you wish to use with your bot.
  export BUCKET=<NAME OF BUCKET TO STORE KEYS>
  export BOT=<NAME OF YOUR BOT>
  ```
+4. Install dependencies and link using npm i and npm link
```

Comment:
also a nit, I trend to put commands in back ticks, `npm i` and `npm link`, so that they jump out as commends.

Reply:
sounds good!